presto-native-execution/src/test/java/com/facebook/presto/spark/AbstractTestPrestoSparkQueries.java

```diff
@@ -115,7 +115,7 @@ private void setupJsonFunctionNamespaceManager(QueryRunner queryRunner)
                 ImmutableMap.of(
                         "supported-function-languages", "CPP",
                         "function-implementation-type", "CPP",
-                        "json-based-function-manager.path-to-function-definition", "src/test/resources/eq.json"));
+                        "json-based-function-manager.path-to-function-definition", "src/test/resources/functions.json"));
```

Comment:
If you rework on this PR, can we piggyback renaming "functions.json" with a better qualifier that makes better sense to people not actively working in this space? "native_functions.json" or "external_functions.json"?

Reply:
Sure. I'll update the file name to `external_functions.json`